docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
+description: Learn more about searching and broadcasting available plugins for Porter
+---
+
+## Search
+
+Porter maintains a list of plugins that users can search via
+`porter plugin search [NAME]`. If no name is supplied, the full listing will be
+returned.  See the help menu for all command options: `porter plugin search -h`.
+
+For example, here we search for an `azure` plugin:
```

Comment:
"here we search for a plugin with the term 'azure' in the name", just to make it clear that it's not an exact match